### PR TITLE
Update dev requirements to the latest versions

### PR DESCRIPTION
**Pull request by lgp171188**
Dev requirements updated to the latest versions listed in https://requires.io/github/mozilla/fjord/requirements/?branch=master

### CODE REVIEW

File: requirements/dev.txt [hunk 1]
```diff
@@ -30,8 +30,8 @@ ordereddict==1.1
 # If you are updating this, you might want to update
 # the hashes of the hooks repositories listed in the
 # .pre-commit-config.yaml file as well.
-# sha256: AnUshRna3Tz1V6c2KD9KB3LF72xU6XwRPs3_1mtnrKw
-pre-commit==0.5.5
+# sha256: wapVuGj_cYauuuwMQDlL2MHHiGdXK3J-WpxzS56oL9I
+pre-commit==0.6.2
```
Comment (willkg) [inline requirements/dev.txt]: When we update pre-commit, do we also have to update the sha in `.pre-commit-config.yaml`?

**Comment by willkg** [comment]
I had one minor note about pre-commit, but otherwise this looks good to me.

File: requirements/dev.txt (hunk 1 above)
Comment (lgp171188) [inline requirements/dev.txt]: There are no newer releases in the pre-commit-hooks (https://github.com/pre-commit/pre-commit-hooks) repository. So we are good.

**Comment by willkg** [comment]
Awesome! Thank you!

**Pull request merged**